nightly e2e test failure: `TestE2EMefEline::test_030_EVC_path_disjointness`

Looked like a transient type of error, however not even reruns solved it, and this test case landed recently, so let's dive into it to understand what happened:

```
    def wait_switches_connect(self):
        max_wait = 0
        while any(not sw.connected() for sw in self.net.switches):
            time.sleep(1)
            max_wait += 1
            if max_wait > 30:
                status = [(sw.name, sw.connected()) for sw in self.net.switches]
>               raise Exception('Timeout: timed out waiting switches reconnect. Status %s' % status)
E               Exception: Timeout: timed out waiting switches reconnect. Status [('AL2', True), ('AL3', True), ('Ampath1', False), ('Ampath2', True), ('Ampath3', True), ('Ampath4', True), ('Ampath5', True), ('Ampath7', True), ('JAX1', True), ('JAX2', True), ('SanJuan', True), ('SoL2', True)]


------------------------------- start/stop times -------------------------------
=========================== rerun test summary info ============================
RERUN tests/test_e2e_14_mef_eline.py::TestE2EMefEline::test_030_EVC_path_disjointness
RERUN tests/test_e2e_14_mef_eline.py::TestE2EMefEline::test_030_EVC_path_disjointness
=========================== short test summary info ============================
ERROR tests/test_e2e_14_mef_eline.py::TestE2EMefEline::test_030_EVC_path_disjointness - Exception: Timeout: timed out waiting switches reconnect. Status [('AL2', True), ('AL3', True), ('Ampath1', False), ('Ampath2', True), ('Ampath3', True), ('Ampath4', True), ('Ampath5', True), ('Ampath7', True), ('JAX1', True), ('JAX2', True), ('SanJuan', True), ('SoL2', True)]
```

--- a/tests/test_e2e_14_mef_eline.py
+++ b/tests/test_e2e_14_mef_eline.py
@@ -363,6 +363,7 @@ class TestE2EMefEline:
         for current, backup in zip(current_path, backup_path):
             assert current["endpoint_a"]["id"] == backup["endpoint_a"]["id"]
             assert current["endpoint_b"]["id"] == backup["endpoint_b"]["id"]
+        self.net.reconnect_switches()
 
     def test_030_EVC_path_disjointness(self):
         """Testing disjointness by expecting a specific failover_path."""
@@ -396,3 +397,4 @@ class TestE2EMefEline:
         ]
         assert len(failover_path) == len(expected_failover_path)
         assert failover_path == expected_failover_path
+        self.net.net.configLinkStatus('Ampath1', 'SoL2', 'up')

